Hexapod/v1/default.yaml

```diff
@@ -0,0 +1,42 @@
+camera_config:
+  elevation_coeffs:
+    # Fit of https://github.com/bxin/hexrot/blob/master/LUT/Camera%20Hexapod%20Motions%20in%20Elevation%20Axis%202020%2007%2023.xlsx
+    # commit 0513b27 2020-08-04
+    - [0]
+    - [6.41754532e+02, 5.52880420e-01, -2.06081416e-01, 1.14135938e-04, 3.68377168e-06]
+    - [-6.05001038e+02, 1.58266699e+01, 4.76907398e-03, -1.00806469e-03, 2.63260121e-06]
+    - [1.77291858e-02, -2.16979613e-07, -5.39973300e-06, -1.50534189e-10, 1.41927682e-10, -7.00491997e-14, -9.63132987e-16]
+    - [0]
+    - [0]
+  temperature_coeffs:
+    # From Document 7334 page 9 last modified 2016-01-20: z correction is 1.32mm/20C
+    - [0]
+    - [0]
+    - [0, 66.0]
+    - [0]
+    - [0]
+    - [0]
+  min_temperature: -20
+  max_temperature: 20
+m2_config:
+  elevation_coeffs:
+    # Fit of https://github.com/bxin/hexrot/blob/master/LUT/M2%20Hexapod%20Motions%20in%20Elevation%20Axis%202020%2007%2023.xlsx
+    # commit 0513b27 2020-08-04
+    - [0]
+    - [5.56668905e+02, 7.65797334e-01, -1.77760096e-01, -8.68218063e-05, 5.59368293e-06, -1.05061161e-08]
+    - [-6.56964062e+02, 1.80665659e+01, -1.05003277e-02, -9.50679179e-04, 9.28818285e-07, 8.94682824e-09]
+    - [6.70511777e-03, -3.46721579e-06, -1.99375935e-06, -3.41214391e-10, 6.23384895e-11, -1.24237443e-13]
+    - [0]
+    - [0]
+  temperature_coeffs:
+    # From Document 7334 page 9 last modified 2016-01-20: z correction is 1.05mm/20C
+    #
+    - [0]
+    - [0]
+    - [0, 52.5]
```

Comment:
This should negative because when T goes higher we want to move M2 and the camera closer to M1M3.